ops/framework.py

```diff
@@ -903,6 +911,17 @@ def _emit(self, event: EventBase):
             # Again, only commit this after all notices are saved.
             self._storage.save_notice(event_path, observer_path, method_name)
         if saved:
+            from . import tracing  # break circular import
+
+            if tracing:
+                # When an app has just been deployed, the tracing relation is not yet established.
+                # The operator would like to see the tracing data for early events like install.
```

Comment:
Nit: I think "operator" is confusing when used to describe the user, because that's what we call ops.

```suggestion
                # The user would like to see the tracing data for early events like install.
```

Or "Juju user" or "charmer".

Reply:
rewritten